### PR TITLE
chore(console): polish UI and add ac profile loading skeleton

### PR DESCRIPTION
**Pull request by charIeszhao**
## Summary

- Polish AC profile UI and add loading skeleton.
- Fix a minor issue that email does not refresh aftering changing it successfully

image

image

## Checklist

- [ ] `.changset-staged`
- [ ] unit tests
- [ ] integration tests
- [ ] docs

OR

- [x] This PR is not applicable for the checklist

### CODE REVIEW

File: packages/console/src/components/UserInfoCard/index.tsx [hunk 1]
```diff
@@ -7,23 +7,24 @@ import * as styles from './index.module.scss';
 
 type Props = {
   className?: string;
-  user: Partial<
+  user?: Partial<
```
Comment (gao-sun) [inline packages/console/src/components/UserInfoCard/index.tsx:10]: just double check, does this pr have a screenshot of the testing result? ok if no for this one though - it'll be great to have one for ui-related changes

File: packages/console/src/containers/AppContent/components/UserInfo/index.tsx [hunk 2]
```diff
@@ -39,7 +39,7 @@ const UserInfo = () => {
     update,
   } = useUserPreferences();
 
-  if (!user) {
+  if (isLoadingUser) {
```
Comment (gao-sun) [inline packages/console/src/containers/AppContent/components/UserInfo/index.tsx:42]: did we have alignment of using `isLoading` vs `!data`?

File: packages/console/src/pages/Profile/index.tsx [hunk 3]
```diff
@@ -1,79 +1,98 @@
...
-                value: passwordEncrypted,
-                renderer: (value) => (value ? <span>********</span> : <NotSet />),
-                action: {
-                  name: 'profile.change',
-                  handler: () => {
-                    navigate('verify-password', {
-                      state: { email: primaryEmail, action: 'changePassword' },
-                    });
+      {showLoadingSkeleton && <Skeleton />}
+      {!!user && !showLoadingSkeleton && (
```
Comment (gao-sun) [inline packages/console/src/pages/Profile/index.tsx:52]: nit: just `user`?

**Review by gao-sun — APPROVED** [review]

File: packages/console/src/components/UserInfoCard/index.tsx (hunk 1 above)
Comment (charIeszhao) [reply packages/console/src/components/UserInfoCard/index.tsx:10]: Yes, this was funny. I actually prepared a screenshot when I was drafting the PR, but then I accidentally copied something else and lost the screenshot in pasteboard. Then I was like, "F it. I'm going to create the PR anyway..."
So...

lol, I'm updating screenshots right now

File: packages/console/src/containers/AppContent/components/UserInfo/index.tsx (hunk 2 above)
Comment (charIeszhao) [reply packages/console/src/containers/AppContent/components/UserInfo/index.tsx:42]: Yes, I think we did. `isLoading` is equivalent to `!data && !error`, while `!data` doesn't care about if the request has ended up with some error, in which case we should not show the loading.

File: packages/console/src/pages/Profile/index.tsx (hunk 3 above)
Comment (charIeszhao) [reply packages/console/src/pages/Profile/index.tsx]: Alright that'll work, too. Updated

File: packages/console/src/components/UserInfoCard/index.tsx (hunk 1 above)
Comment (gao-sun) [reply packages/console/src/components/UserInfoCard/index.tsx:10]: lol thank you old tea

**Pull request merged**